src/settings_translation_file.cpp

```diff
@@ -213,7 +213,7 @@ fake_function() {
 	gettext("Fall bobbing");
 	gettext("Multiplier for fall bobbing.\nFor example: 0 for no view bobbing; 1.0 for normal; 2.0 for double.");
 	gettext("3D mode");
-	gettext("3D support.\nCurrently supported:\n-    none: no 3d output.\n-    anaglyph: cyan/magenta color 3d.\n-    interlaced: odd/even line based polarisation screen support.\n-    topbottom: split screen top/bottom.\n-    sidebyside: split screen side by side.");
```

Comment:
If you want to automatically update the other files as well, feel free to do it, but its best if you used the automatic method: exchange `false` with `true` in builtin/mainmenu/tab_settings.lua lines 727 and 735. And then open the client (main menu is enough). Then dont forget to change it back to false. But note, this is not a must have, you can chose to not do it, then it will be done at some later point by sb else :)